docs/contributing/test-and-docs.md

```diff
@@ -0,0 +1,234 @@
+### Run tests and test documentation
```

Comment:
Can you remove the `and test documentation` here, and probably rename the file (and links to this file)?